core/src/main/java/net/sourceforge/jnlp/util/lockingfile/WindowsLockedFile.java

```diff
@@ -0,0 +1,29 @@
+package net.sourceforge.jnlp.util.lockingfile;
+
+import java.io.File;
+import java.io.IOException;
+
+public class WindowsLockedFile extends LockedFile {
```

Comment:
`WindowsLockedFile` simply skips the calls to `lockProcess()`. If we are going to refactor the whole package we could simply remove `WindowsLockedFile` and change `lockProcess()` do to nothing on Windows.